templates/deployments.yaml

```diff
@@ -447,12 +466,25 @@ spec:
         - name: st2-config-vol
           mountPath: /etc/st2/st2.user.conf
           subPath: st2.user.conf
+        {{- if .Values.secrets.st2.datastore_crypto_key }}
```

Comment:
Following the https://github.com/StackStorm/st2-packages/blob/e86adb5e75b5013dac43198f9e464b29cb08ed58/scripts/st2bootstrap-deb.sh#L356-L360 I don't think `st2rulesengine` needs access to st2 datastore crypto key.

@m4dcoder can you confirm?

Reply:
`st2rulesengine` doesn't need access to st2 datastory crypto key. Please remove it.